app/build.gradle

```
@@ -142,5 +142,5 @@ android {
 }
 
 dependencies {
-    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:1.3.1'
+    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:1.3.2'
```

`2.0.1` is out already. I'd recommend using the latest.

```suggestion
    implementation 'com.google.androidbrowserhelper:androidbrowserhelper:2.0.1'
```

Andre

---

Thank you for the suggestion ! 

I tried and it implies for me to fix a few more errors that I couldn't fix easily as I lack android studio expertise. (related to Theme.appcompat not found ~)

Therefore, I'll make sure we use the latest version later and go with the 1.3.2 for now.

Thank you again ! 
Gireg

---

Gotcha - created #3 with the necessary changes.